### PR TITLE
docs: Add user and tech documentation for geo steps TCTC-4332

### PR DESCRIPTION
**Pull request by lukapeschke**

### CODE REVIEW

File: docs/_docs/tech/steps.md [hunk 1]
```diff
@@ -1720,6 +1803,60 @@ The following operators are supported by the formula step (note that a value can
 | Label 2 | 1      | 13     | 7      | 3       | -4     |
 | Label 3 | 5      | 20     | 5      | 2       | 1      |
 
+### `hierarchy` step
+
+Hierarchy for geographical data.
+
+This step dissolves data for every hierarchy level, and adds a hierarchy level column containing a level
+(with 0 being the lowest granualrity, i.e. the highest level).
```
Comment (davinov) [inline docs/_docs/tech/steps.md:1811]: granualrity > granularity

File: docs/_docs/tech/steps.md [hunk 2]
```diff
@@ -1135,6 +1135,89 @@ Delete a column.
...
+
+```javascript
+{
+    name: 'dissolve',
+    groups: ['my-column', 'some-other-column'],
+    include_nulls: true,
+}
+```
+
+**This step is supported by the following backends:*
```
Comment (davinov) [inline docs/_docs/tech/steps.md:1150]: misses a star at the end for the bold formatting

File: docs/_docs/user-interface/simplify.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+permalink: /docs/simplify/
+---
+
+### Geographical simplification
+
+Use this step in case you want to simplify heavy geographical data.
+
+When simplifying your data, every point that is closer than a specific distance to the previous one is suppressed.
+This step can be useful if you have a very precise shape for a country (such as one-meter precision), but want to quickly
+draw a map chart. In that case, you may want to *simplify* your data.
```
Comment (davinov) [inline docs/_docs/user-interface/simplify.md:12]: Not easy to explain! Maybe we can link to https://geopandas.org/en/stable/docs/reference/api/geopandas.GeoSeries.simplify.html ?

File: docs/_docs/user-interface/simplify.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+- `Geo` widget
+- Search bar
+
+#### Options reference
+
+<img src="../../img/docs/user-interface/simplify_step_form.jpg" width="350" />
+
+- `Tolerance`: The tolerance to use when simplifying your data. After simplification, no points will be closer than
+  `Tolerance`. The unit depends on the unit of your data's projection, but in general, it's expressed in meters.
```
Comment (davinov) [inline docs/_docs/user-interface/simplify.md:28]: Depends on the projection also? (not very important though, people will probably try and see)

**Review by davinov — APPROVED** [review]

File: docs/_docs/user-interface/simplify.md (hunk 3 above)
Comment (lukapeschke) [reply docs/_docs/user-interface/simplify.md:12]: I was hesitating, but it seems complex for user docs... Maybe I can move this paragraph to the tech docs and just add a simpler explanation here ? Something like _This step reduces the size of your data by making it less precise. The higher you set the tolerance, the smaller (and the less precise) your data will be. See the  **technical documentation** for details_

File: docs/_docs/user-interface/simplify.md (hunk 4 above)
Comment (lukapeschke) [reply docs/_docs/user-interface/simplify.md:28]: Maybe I can add _for GIS projections_ ?

File: docs/_docs/user-interface/simplify.md (hunk 3 above)
Comment (davinov) [reply docs/_docs/user-interface/simplify.md:12]: Yes, it's simpler!

**Pull request merged**